src/boost_fractor.jl

```diff
@@ -66,10 +68,71 @@ mutable struct SetupBoundaries
     eps::Array{Complex{Float64},1}   # = [1,9,1]
     relative_tilt_x # = [0,0]
     relative_tilt_y # = [0,0]
-    relative_surfaces::Array{Complex{Float64},3} # = [z, x,y ]
+    relative_surfaces::Array{Complex{Float64},3} # = [z, x, y]
     # etc.
 end
 
```

Comment:
Here it is possible to define two non compatible setups for disk geometries (using relative_tilt, relative_surfaces in SetupBoundaries and surface in Phase_shifts). This is an issue and may have to be adressed.

Reply:
I am not quite sure what you mean, do you mean that one can make a relative_surfaces that already contains plate tilt or that surface in Phase_shifts conflicts with the arrays from SetupBoundaries? The Phase shifts are meant to be calculated from the Boundaries (the idea was that SetupBoundaries still describes the experimental setup, Phase_shifts is just supposed to bring the values form it into a form that is more convenient for the simulation).

Reply:
Ok, then we need to make sure, that normal users do not manually use these helper functions. One thing, I wouldnt export SeedPhaseShifts then (btw. you didn't change the name up there anyways). Secondly I'd like a comment somewhere explicity saying Do not use for initialization or similar.